addons/project/static/tests/tours/project_update_tour_tests.js

```diff
@@ -165,7 +165,7 @@ registry.category("web_tour.tours").add('project_update_tour', {
     trigger: ".o_list_button_add",
     content: "Make sure the milestone is saved before continuing",
 }, {
-    trigger: "td[data-tooltip='New milestone'] + td",
+    trigger: "td[data-tooltip='New milestone'] ~ td:contains('Dec 12')",
```

Comment:
FYI-When the sale_project module is installed, the created project is billable, so a **Sales Order Item** field is added between the Name and Deadline. Therefore, I need to get the specific td containing the deadline date, rather than simply selecting the next td.